tests/core/v5_1/test_v51_rpc_handlers.py

```diff
@@ -217,11 +217,14 @@ async def test_v51_rpc_set_enr_web3(make_request, w3, new_enr):
 
 
 @pytest.mark.trio
-async def test_v51_rpc_set_enr_web3_duplicate(make_request, w3, new_enr):
+async def test_v51_rpc_set_enr_web3_with_invalid_sequence_number(
+    make_request, w3, new_enr
+):
+    new_enr._sequence_number = 2
```

Comment:
Monkeypatching the sequence number seems like a bad approach.  You can do this cleaner via `ENRFactory(sequence_number=XXX)`  If you want to be a little more flexible you can do something like `ENRFactory(sequence_number=secrets.randbelow(100) + 1)`